### PR TITLE
Brenkt-SK851-TRANS-WEB-Domain-Errors 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/domains/domain_errors/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+## Requisitos
+
+- Tener uno o más dominios.
+- Haber iniciado sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
+- Estar actualizado en los [pagos](https://docs.ovh.com/es/billing/gestionar-facturas-ovh/#pay-bills) y [renovaciones](https://docs.ovh.com/es/billing/renovacion-automatica-ovh/#renewal-management) de los servicios asociados (dominio y alojamiento web).
+
+## Procedimiento
+
+Acceda a la sección [Web Cloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) y seleccione `Dominios`{.action} en el área de cliente de OVHcloud` `{.action}. Haga clic en `Operaciones en curso`{.action} sobre la lista de dominios.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.es-es.md]: empty quotes

File: pages/web/domains/domain_errors/guide.es-es.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+![Dominio](images/domain-error06.png){.thumbnail}
+
+#### Error en los servidores DNS
+
+Si los servidores DNS asociados a un dominio no funcionan, puede producirse un error.<br>
+En el ejemplo a continuación, la dirección IP del servidor DNS no responde.
+
+![Dominio](images/domain-error07.png){.thumbnail}
+
+En la sección `Dominios`{.action}, seleccione el dominio correspondiente y abra la pestaña `Servidores DNS`{.action}. Desde esta pestaña, [modifique sus servidores DNS](https://docs.ovh.com/fr/domains/generalites-serveurs-dns/).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.es-es.md]: FR link

File: pages/web/domains/domain_errors/guide.it-it.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+![dominio](images/domain-error-table02.png){.thumbnail}
+
+### Esempi
+
+> [!primary]
+>
+> L'elenco degli esempi sotto riportato non è esaustivo. Se riscontri un errore non specificato in questa guida:
+>
+> - Verifica di essere aggiornato sui [pagamenti](https://docs.ovh.com/it/billing/gestire-fatture-ovhcloud/#pay-bills) e sui [rinnovi](https://docs.ovh.com/it/billing/imposta_il_rinnovo_automatico_dei_tuoi_servizi_ovh/#renewal-management) dei tuoi domini.
+> - Per agire, clicca sul pulsante...``{.action} a destra dell'operazione in questione.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.it-it.md]: Misplaced quotes

File: pages/web/domains/domain_errors/guide.it-it.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+> - Leggi il messaggio descrittivo e controlla se ti permette di risolvere l'errore.
+>
+> Se, nonostante queste verifiche, non sei in grado di agire sul dominio, ti invitiamo ad aprire un ticket di assistenza dal tuo Spazio Cliente OVH.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.it-it.md]: OVH > OVHcloud

File: pages/web/domains/domain_errors/guide.it-it.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+>
+
+#### Richiesta di documenti
+
+Alcune estensioni di domini richiedono una documentazione per giustificarne l'utilizzo. In questo caso, trasmetterai i documenti dalla finestra `Operazioni in corso`{.action}.
+
+![dominio](images/domain-error01.png){.thumbnail}
+
+Per fornire il(i) documento(i) necessario(i), clicca sul pulsante...`.`{.action} a destra dell'operazione in questione.<br>
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.it-it.md]: Please fix format

File: pages/web/domains/domain_errors/guide.pl-pl.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+**Dowiedz się, jak postępować w przypadku wystąpienia błędu dla domeny.**
+
+## Wymagania początkowe
+
+- Posiadanie jednej lub kilku domen
+- Zalogowanie do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl)
+- Aktualizacja w [płatności](https://docs.ovh.com/pl/billing/zarzadzanie-fakturami-ovhcloud/#pay-bills) i [odnowienie](https://docs.ovh.com/pl/billing/przewodnik_dotyczacy_opcji_automatycznego_odnawiania_uslug_w_ovh/#renewal-management) powiązanych usług (domena i hosting)
+## W praktyce
+
+W [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) przejdź do sekcji `Web Cloud`{.action}, a następnie `nazwy domen`{.action}. Kliknij `Operacje w toku`{.action} nad listą domen.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.pl-pl.md]: Domains is "Domeny" in PL in the Control Panel

File: pages/web/domains/domain_errors/guide.pl-pl.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+- `Domena`: Domena, której dotyczy operacja.
+- `Operacja`:  Trwa wykonywanie operacji dla domeny.
+- `Komentarz`: Szczegóły operacji w trakcie. Instrukcje.
+- `Data przetwarzania`: Data utworzenia operacji.
+- `Data aktualizacji`:  Znacznik czasu aktualizacji operacji w trakcie.
+- `Data zakończenia`: Data zakończenia operacji.
+- `Status`: Obecny stan operacji.
+
+Wszystkie operacje wymienione w tej tabeli nie wymagają interwencji, aby były przeprowadzane w normalnych warunkach.<br>
+W niniejszym przewodniku przyjrzymy się operacjom **z błędem**, posługując się przykładami, które się powtarzają.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.pl-pl.md:42]: Please check if bold phrase should contain also "operacjom"

File: pages/web/domains/domain_errors/guide.pt-pt.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+![domínio](images/domain-error06.png){.thumbnail}
+
+#### Erro associado aos servidores DNS
+
+Pode ocorrer um erro se os servidores DNS associados a um domínio não funcionarem.<br>
+No exemplo abaixo, o endereço IP do servidor DNS não responde.
+
+![domínio](images/domain-error07.png){.thumbnail}
+
+Na secção `Nomes de domínio`{.action}, selecione o domínio em questão e clique no separador `Servidores DNS`{.action}. A partir deste separador, [modifique os seus servidores DNS](https://docs.ovh.com/fr/domains/generalites-serveurs-dns/).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.pt-pt.md]: FR link

File: pages/web/domains/domain_errors/guide.de-de.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+## Voraussetzungen
+
+- Sie besitzen eine oder mehrere Domains.
+- Sie sind in Ihrem [OVHcloud Kundencenter](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.de/&ovhSubsidiary=de) angemeldet.
+- Sie haben keine ausstehenden [Zahlungen](https://docs.ovh.com/de/billing/ovh-rechnungen-verwalten/#pay-bills) und [Verlängerungen](https://docs.ovh.com/de/billing/anleitung_zur_nutzung_der_automatischen_verlangerung_bei_ovh/#renewal-management) der dazugehörigen Dienstleistungen (Domainname und Webhosting).
+
+## In der praktischen Anwendung
+
+Gehen Sie [in Ihrem OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.de/&ovhSubsidiary=de) Kundencenter in den Bereich `Web Cloud`{.action} und `dann Domainnamen`{.action}. Klicken `Sie auf`{.action} Laufende Operationen über der Liste der Domainnamen.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.de-de.md]: Quotes don't contain the proper names

File: pages/web/domains/domain_errors/guide.de-de.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+>
+> Dieser Typ der Sperrung wird von der Registry und nicht von OVHcloud initiiert. Selbst wenn die Domain von der Registry blockiert wird, erscheinen ihre DNS Server als `Active` in Ihrem OVHcloud Kundencenter.
+
+Um zu überprüfen, ob Ihr Domainname derart gesperrt ist, gehen Sie in die Tabelle der `laufenden Operationen`{.action}.
+
+![domain](images/domain-error08.png){.thumbnail}
+
+Um Ihre Domain zu überprüfen, empfehlen wir Ihnen die Verwendung des von der Registry ausgestellten Tools zur Überprüfung:
+
+- Für eine .de-**Domain**: <https://nast.denic.de/>.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.de-de.md]: Please fix bold formatting for both sentences

File: pages/web/domains/domain_errors/guide.de-de.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+![domain](images/domain-error06.png){.thumbnail}
+
+#### Fehler bei den DNS Servern
+
+Es kann ein Fehler auftreten, wenn die DNS Server, die Sie einer Domain anhängen, nicht funktionieren.<br>
+Im folgenden Beispiel antwortet die IP-Adresse des DNS Servers nicht.
+
+![domain](images/domain-error07.png){.thumbnail}
+
+Wählen Sie im Bereich `Domainnamen`{.action} die betreffende Domain aus und klicken Sie dann auf den Tab `DNS Server`{.action}. Ändern Sie in diesem Tab [Ihre DNS Server](https://docs.ovh.com/fr/domains/generalites-serveurs-dns/).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.de-de.md]: FR link

File: pages/web/domains/domain_errors/guide.en-gb.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+> - Check that you are up to date with [payments](https://docs.ovh.com/gb/en/billing/manage-ovh-bills/#pay-bills) and domain name [renewals](https://docs.ovh.com/gb/en/billing/how-to-use-automatic-renewal-at-ovh/#renewal-management).
+> - Check if you can do this by clicking the `...`{.action} button to the right of the operation concerned.
+> - Read the descriptive message and see if it helps you resolve the error.
+>
+> If, despite these checks, you are unable to change the domain name, please open a support ticket via the OVHcloud Control Panel.
+>
+
+#### Document requests
+
+Some domain name extensions require you to justify their use by providing documents. If this is the case, you will need to submit the documents from the `Ongoing`{.action} Operations window.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.en-gb.md]: `Ongoing`{.action} Operations -> `Ongoing operations`{.action}

File: pages/web/domains/domain_errors/guide.en-gb.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+![domain](images/domain-error06.png){.thumbnail}
+
+#### DNS server error
+
+An error may occur if the DNS servers you attach to a domain name are not working.<br>
+In the example below, the IP address of the DNS server is not responding.
+
+![domain](images/domain-error07.png){.thumbnail}
+
+In the `Domain`{.action} names section, select the domain name concerned, then click the `DNS`{.action} Servers tab. In this tab, [edit your DNS](https://docs.ovh.com/fr/domains/generalites-serveurs-dns/) servers.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/domain_errors/guide.en-gb.md]: FR link

File: pages/web/domains/domain_errors/guide.pl-pl.md (hunk 7 above)
Comment (BrenKt) [reply pages/web/domains/domain_errors/guide.pl-pl.md:42]: z błędem = error > this is good

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**